corfu.el

```
@@ -835,6 +845,13 @@ there hasn't been any input, then quit."
   "Return t if a candidate is selected and previewed."
   (and corfu-preview-current (/= corfu--index corfu--preselect)))
 
+
+(defun corfu-insert-separator-char ()
+  "Insert a separator character and inhibit quit on completion boundary.
+Bind to a convenient key in corfu-map."
+  (interactive)
+  (insert corfu-separator-char))
```

Here we should probably throw an error if the separator char is nil?

---

Done (with `nil`).

---

done.